examples/landmarks/at.html

```diff
@@ -281,4 +281,4 @@ <h2 id="id3">Related W3C Documents</h2>
     </script>
     <script type="text/javascript" src="js/SkipTo.min.js"></script>      
   </body>
-</html>
```

Comment:
I was using the web editor, which does actually support EditorConfig.
This diff is actually saying that I fixed the missing eof newline